### PR TITLE
Fix Helm and Add Probe

### PR DESCRIPTION
**Pull request by Ianwu0812**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

Add probe into helm deployment

Add liveness and readiness probe to let kubernetes know that
application can not serve.

Fix kubectl get pods in helm note.
It should be app.kubernetes.io/instance but release.

Some env will occurs error when value is empty. Add quote pipeline
in deployment to fix this problem.

### Benefits

Some value in helm can be empty.

Helm note will show correct infomation.

Kubernetes will restart when application occurs error.

Fix deployment did not use `prom2teams.port` setting from values.

### Possible Drawbacks

No

### Applicable Issues

Same as benefit

### CODE REVIEW

File: helm/templates/deployment.yaml [hunk 1]
```diff
@@ -39,15 +39,25 @@ spec:
...
-            value: {{ .Values.prom2teams.ip }}
+            value: {{ .Values.prom2teams.ip | quote}}
           - name: PROM2TEAMS_CONNECTOR
             value: {{ .Values.prom2teams.connector }}
           - name: PROM2TEAMS_GROUP_ALERTS_BY
-            value: {{ .Values.prom2teams.group_alerts_by }}
+            value: {{ .Values.prom2teams.group_alerts_by | quote}}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          readinessProbe:
```
[the remark this answers is not in the document]
Comment (vicsufer) [reply helm/templates/deployment.yaml:51]: Also at #225 we're working on `/alive` and `/ready` endpoints for livesness and readiness probes respectively. Mind changing the tcp probes for http ones?

**Review by vicsufer — CHANGES_REQUESTED** [review]

**Comment by vicsufer** [comment]
Thanks for taking the time to contribute @Ianwu0812 I've left some comments on your code. 
Also please update the changelog when finish.

**Comment by vicsufer** [comment]
Looks good to me, please update the CHANGELOG.md and we're ready to go.

**Review by vicsufer — APPROVED** [review]

**Review by dortegau — APPROVED** [review]

**Comment by vicsufer** [comment]
Ey @Ianwu0812! We really want to merge this feature into stable branches, can you please update the README.md? Thanks!

**Comment by Ianwu0812** [comment]
Hi @vicsufer, I have changed the CHANGELOG.md, is that fit your requirement?

**Comment by vicsufer** [comment]
@Ianwu0812 I can't see any change commited to CHANGELOG.md
https://github.com/idealista/prom2teams/pull/224/files

**Pull request closed without merging**